package.json

```diff
@@ -30,39 +32,41 @@
     "copy": "npm-run-all copy:*",
     "copy:fonts": "shx cp -R node_modules/videojs-font/fonts dist/font",
     "copy:examples": "shx cp -R docs/examples dist/",
-    "build:js": "rollup -c --no-progress",
+    "build:js": "rollup -c",
     "build:css": "npm-run-all build:css:*",
     "build:css:cdn": "node-sass src/css/vjs-cdn.scss dist/alt/video-js-cdn.css",
     "postbuild:css:cdn": "postcss --verbose --config postcss.config.js -d dist/alt dist/alt/video-js-cdn.css",
     "build:css:default": "node-sass src/css/vjs.scss dist/video-js.css",
     "postbuild:css:default": "postcss --verbose --config postcss.config.js -d dist/ dist/video-js.css",
     "build:lang": "npm-run-all build:lang:*",
     "build:lang:js": "vjslang --dir dist/lang",
-    "build:lang:copy": "shx --verbose cp -R lang/* dist/lang/",
+    "build:lang:copy": "shx cp -R lang/* dist/lang/",
     "minify": "npm-run-all minify:*",
     "minify:js": "babel-node build/minify.js",
     "minify:css": "npm-run-all minify:css:*",
     "minify:css:cdn": "cleancss dist/alt/video-js-cdn.css -o dist/alt/video-js-cdn.min.css",
     "minify:css:default": "cleancss dist/video-js.css -o dist/video-js.min.css",
     "watch": "npm-run-all -p watch:*",
     "watch:lang": "chokidar --initial 'lang/**/*.json' -c 'npm run build:lang'",
-    "watch:rollup": "rollup -c -w ",
+    "watch:rollup": "rollup -c -w --no-progress",
     "watch:css-default": "npm run build:css:default -- --watch 'src/**/**/*.scss'",
     "watch:css-cdn": "npm run build:css:cdn -- --watch 'src/**/**/*.scss'",
     "assets": "node build/assets.js",
     "lint": "vjsstandard",
     "lint-errors": "vjsstandard --errors",
     "karma-server": "karma start test/karma.conf.js --singleRun=false --auto-watch",
-    "test": "npm-run-all build-dev test:*",
+    "pretest": "npm run build-dev",
+    "test": "npm-run-all -p test:*",
     "test:node-require": "node test/require/node.js",
     "test:a11y": "node build/test-a11y.js",
     "test:unit": "karma start test/karma.conf.js",
     "docs": "npm-run-all clean docs:lint docs:api",
     "docs:api": "jsdoc -c .jsdoc.json",
     "postdocs:api": "node ./build/fix-api-docs.js",
-    "netlify": "babel-node ./build/netlify-docs.js",
     "docs:lint": "remark -- './{,!(node_modules)/**/}!(CHANGELOG)*.md'",
     "docs:fix": "remark --output -- './{,!(node_modules)/**/}!(CHANGELOG)*.md'",
+    "docs:lang": "node build/translations.js",
```

Comment:
we could have this run via lint-staged, whenever translations are changed

Reply:
Yeah, maybe. Let's do it in another PR, though.